src/api/app/controllers/webui/groups_controller.rb

```diff
@@ -55,7 +55,6 @@ def group_params
   end
 
   def set_group
-    required_parameters :title
```

Comment:
In the commit message you mean:

Move `title` **to** route.

And you forgot the part of the route. It should be in this commit.

Reply:
Done